sphinx/transforms/__init__.py

```diff
@@ -343,21 +343,5 @@ def get_tokens(self, txtnodes):
         texttype = {True: 'literal',  # "literal" text is not changed:
                     False: 'plain'}
         for txtnode in txtnodes:
-            nodetype = texttype[isinstance(txtnode.parent,
-                                           (nodes.literal,
-                                            addnodes.literal_emphasis,
-                                            addnodes.literal_strong,
-                                            addnodes.desc_addname,
-                                            addnodes.desc_annotation,
-                                            addnodes.desc_name,
-                                            addnodes.desc_optional,
-                                            addnodes.desc_parameter,
-                                            addnodes.desc_parameterlist,
-                                            addnodes.desc_signature_line,
-                                            addnodes.desc_type,
-                                            addnodes.production,
-                                            nodes.math,
-                                            nodes.image,
-                                            nodes.raw,
-                                            nodes.problematic))]
-            yield (nodetype, txtnode.astext())
+            smartquotable = not is_smartquotable(txtnode)
```

Comment:
@tk0miya naming is confusing here, 

```
notsmartquotable = not is_smartquotable(txtnode)
yield (texttype[notsmartquotable], txtnode.astext())
```
would be clearer...

Reply:
Absolutely

Reply:
Done at 0059c05a8